tiny_dnn/core/kernels/conv2d_grad_op_avx.h

```diff
@@ -439,8 +485,9 @@ void avx_conv2d_5x5_back_kernel_one(const core::conv_params& params,
 
     // accumulate db
     if (params.has_bias) {
-        //fvec_t& db = *in_grad[2];
-        
+#if 1
```

Comment:
I prefer to use comments & github history. ```#if 1 - #else - #endif``` block is unclear. Do we need to leave this else block?